src/Options.php

```diff
@@ -701,6 +701,28 @@ public function setMaxRequestBodySize(string $maxRequestBodySize): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    /**
+     * Gets the type serializers.
```

Comment:
Please change the description to: `Gets the callbacks used to customize how objects are serialized in the payload of the event.`